templates/AureliaSpa/webpack.config.js

```diff
@@ -0,0 +1,64 @@
+var isDevBuild = process.argv.indexOf('--env.prod') < 0;
+var path = require('path');
+var webpack = require('webpack');
+var AureliaWebpackPlugin = require('aurelia-webpack-plugin');
+var srcDir = path.resolve('./ClientApp');
+var rootDir = path.resolve();
+var outDir = path.resolve('./wwwroot/dist');
+var baseUrl = '/';
+var project = require('./package.json');
+var aureliaModules = Object.keys(project.dependencies).filter(dep => dep.startsWith('aurelia-'));
```

Comment:
Good point. However this approach is no longer used in the current version anyway - dependencies are now listed explicitly.